### PR TITLE
Add zhangxupeng0 as approver

### PR DESCRIPTION
**Pull request by zhangxupeng0**
add a parameter of tide and add a approver

### CODE REVIEW

File: starter-s3.yaml [hunk 1]
```diff
@@ -528,6 +529,7 @@ spec:
         - --history-uri=s3://tide/tide-history.json
         - --github-app-id=$(GITHUB_APP_ID)
         - --github-app-private-key-path=/etc/github/cert
+        - --v=4
```
Comment (Jeffwan) [inline starter-s3.yaml]: Did you already apply this change to Kubernetes in volcano engine ?

Comment (Jeffwan) [reply starter-s3.yaml]: Since this is just for testing purpose. Let's do not check in this change

Comment (zhangxupeng0) [reply starter-s3.yaml]: I did not do anything in volcano engine.And this change updates log level and approvers.

**Review by Jeffwan — APPROVED** [review]

**Pull request merged**